packages/components/src/utils/colors-values.js

```diff
@@ -31,7 +31,8 @@ export const G2 = {
 		700: '#757575', // Meets 4.6:1 text contrast against white.
 		600: '#949494', // Meets 3:1 UI or large text contrast against white.
 		400: '#ccc',
-		200: '#ddd', // Used for most borders.
+		300: '#ddd', // Used for most borders.
+		200: '#e0e0e0',
```

Comment:
I'm not sure about this change. These colors originally did not correspond with the colors in the Sass variables. This change aligns them, but maybe they weren't meant to be aligned in the first place?